web3.md

```diff
@@ -125,5 +129,34 @@ module WEB3
            <balance> ACCTBALANCE </balance>
          ... </account>
 
+    syntax KItem ::= "#eth_getStorageAt"
+ // -------------------------------
+    rule <k> #eth_getStorageAt ... </k>
+         <params> [ (DATA => #parseHexWord(DATA)), QUANTITY:Int, _ ] </params>
```

Comment:
QUANTITY should be a hex string. We will have to revisit `eth_getStorageAt`, as its parameters can involve keccak encodings. https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getstorageat

Reply:
the work involving keccak actually happens on the other side, we shouldn't need to worry about it.

Reply:
But we would have to decode it when it comes in, or I am looking at this the wrong way?